CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+## [1.19.1] - 2020-11-06
+
+### Updated
+
+- Using the latest version of `govuk-frontend`.
+- Upgrading to the latest version of `bootstrapjs`, trying and failing to avoid a vulnerability but still worth upgrading.
```

Comment:
Backstopjs not bootstrapjs 🙂

Reply:
Haha, I was so focussed on the presence of `js` at the end that I missed that.